auth_app/repositories/tokens.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional
+from uuid import UUID
+
+from sqlalchemy import insert, select, update
+
+from auth_app.models.tokens import refresh_tokens
+from auth_app.repositories.base import BaseRepo
+from auth_app.schemes.tokens import (
+    CreateRefresh,
+    GetRefresh,
+    UpdateRefresh,
+)
+
+
+class TokenRepo(BaseRepo):
+
+    async def create_refresh(self, create_data: CreateRefresh) -> GetRefresh:
+        data = create_data.model_dump()
+        stmt = (
+            insert(refresh_tokens).values(**data).returning(refresh_tokens.c)
+        )
+        result = await self.session.execute(stmt)
+        await self.session.commit()
+        row = result.mappings().one()
+        return GetRefresh.model_validate(row)
```

Comment:
Аналогично надо и репозитории переделать
```suggestion
async def create_refresh(self, create_data: CreateRefresh) -> GetRefresh:
    obj = RefreshToken(**create_data.model_dump())
    self.session.add(obj)
    await self.session.commit()
    await self.session.refresh(obj)
    return GetRefresh.model_validate(obj)
```
такой вариант предпочтительнее, так как он намного проще
также у тебя везде репозитории возвращают пайдантик модели, а так будут возвращаться объекты из бд. С точки зрения логики это более корректно

Reply:
исправил

Reply:
нет, не исправил)

Reply:
сделал через add